### PR TITLE
Create LeMonde.fr.xml

### PR DESCRIPTION
**Pull request by Bisaloo**
Preparing myself for travis' wrath...

### CODE REVIEW

**Comment by J0WI** [comment]
Please merge this with `Le_Monde.fr.xml`.

**Comment by Bisaloo** [comment]
I added the securecookie from `Le_Monde.fr.xml`. The targets are already listed in my ruleset, either as comments or as targets.

Would you like me to add the comments on CDN buckets as well?

File: src/chrome/content/rules/LeMonde.fr.xml [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+	<target host="sport.lemonde.fr" />
+	<target host="sports-us.lemonde.fr" />
+	<target host="tahitivahine.lemonde.fr" />
+	<target host="www.tahitivahine.lemonde.fr" />
+	<target host="tennis.lemonde.fr" />
+	<target host="vins.lemonde.fr" />
+	<target host="voile.lemonde.fr" />
+	<target host="volley.lemonde.fr" />
+
+	<securecookie host="^\.?(?:abo|boutique|monabo)\.lemonde\.fr$" name=".+" />
```
Comment (J0WI) [inline src/chrome/content/rules/LeMonde.fr.xml]: You don't have a target for monabo.

**Comment by J0WI** [comment]
CDN buckets are covered in `Lemde.fr.xml`, you may update it in a separate PR.

File: src/chrome/content/rules/LeMonde.fr.xml (hunk 1 above)
Comment (Bisaloo) [inline src/chrome/content/rules/LeMonde.fr.xml]: `pubs.lemonde.fr` is 410 gone. Do you have a working test URL for this subdomain?

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**